lib/Common/CommonDefines.h

```diff
@@ -168,6 +168,7 @@
 #define SYSINFO_IMAGE_BASE_AVAILABLE 1
 #define ENABLE_CONCURRENT_GC 1
 #define ENABLE_ALLOCATIONS_DURING_CONCURRENT_SWEEP 1 // Only takes effect when ENABLE_CONCURRENT_GC is enabled.
+#define ALLOCATIONS_DURING_CONCURRENT_SWEEP_HEAPBLOCK_THRESHOLD 5000 // Number of blocks a heap bucket needs to have before allocations during concurrent sweep feature kicks-in.
```

Comment:
Yes, Oz, it would be great if we can get the SLIST work done sooner. Let's sync up on that offline.

---
In reply to: [148497285](https://github.com/Microsoft/ChakraCore/pull/4104#discussion_r148497285) [](ancestors = 148497285)